internal/db/start/start.go

```diff
@@ -62,7 +62,7 @@ func NewContainerConfig() container.Config {
 EOF
 `},
 	}
-	if utils.Config.Db.MajorVersion >= 14 {
+	if utils.Config.Db.MajorVersion >= 13 {
```

Comment:
We only support PG13+ for the CLI, so this is always `true`. But if it ain't broke... Note that for PG13 the default `config_file` is different: `/var/lib/postgresql/data/postgresql.conf`

Reply:
Hmm I probably broke it after resetting entrypoint. Let me revert this and make another fix